src/stylesheets/elements/_links.scss

```diff
@@ -0,0 +1,7 @@
+.sds-icon-link{
+    @include u-display('table');
+    align-items:center;
```

Comment:
Does  @include u-flex('align-center')  work?

Reply:
It does, I did not want to use if because it is flex even though it produces the right code it feels odd. I've gone ahead and made the change

Reply:
Shouldn't we set display to flex then?

Reply:
Flex is a lot more code as it requires divs around the elements. As it is (display taqble-cell) we can add this class to the link. 
if we use flex we have a wrapper then two cols and links within each (since and "a" cannot span over divs)

Reply:
`<a iconlink> icon text</a>`
vs
        `<div iconlink> 
                <div col>
                       <a>Icon</a>
                </div>
                <div col>
                       <a>text</a>
                </div>
         </div>`